apps/frisc/test/regfile.fil

```diff
@@ -1,18 +1,18 @@
-import "../src/regfile.fil";
+import "frisc/src/regfile.fil";
 
-comp main<'G:1>(
+comp main<G:1>(
   clk:1,
-  reset: ['G,'G+1] 1,
-  en: ['G,'G+1] 1,
-  rd: ['G,'G+1] 5,
-  data: ['G,'G+1] 32,
-  rin1: ['G,'G+1] 5,
-  rin2: ['G,'G+1] 5
+  @[G,G+1] reset:1,
```

Comment:
This seems like old syntax that won't compile. Can we add the `frisc` tests to the test suite?